### PR TITLE
Enhancement: add nodejs and npm

### PR DESCRIPTION
**Pull request by till**
Fixes: easybib/issues#5739

### CODE REVIEW

**Comment by till** [comment]
@fh Tested by Sascha and Yvan. Can you CR?

File: stack-wpt/recipes/role-nodejs.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+node.set['nodejs'] = {
```
Comment (fh) [inline stack-wpt/recipes/role-nodejs.rb]: @till Why are you using this instead of `stack-wpt/attributes/default.rb`?

Comment (till) [inline stack-wpt/recipes/role-nodejs.rb]: @fh Example?

Comment (fh) [inline stack-wpt/recipes/role-nodejs.rb]: https://github.com/till/easybib-cookbooks/blob/master/ies/attributes/default.rb#L1
https://docs.chef.io/attributes.html#attribute-precedence

Comment (till) [inline stack-wpt/recipes/role-nodejs.rb]: I'll refactor them all in a new PR. cc @fh

**Comment by fh** [comment]
As discussed irl: Merging for now, migration to attributes in a second pr with a test eventually

**Pull request merged**